compiler/optimizer/Inliner.cpp

```diff
@@ -5729,7 +5729,7 @@ TR_CallSite::calleeClass()
       TR::StackMemoryRegion stackMemoryRegion(*_comp->trMemory());
 
       int32_t len = _interfaceMethod->classNameLength();
-      char * s = TR::Compiler->cls.classNameToSignature(_interfaceMethod->classNameChars(), len, _comp, stackAlloc);
+      const char *s = TR::Compiler->cls.classNameToSignature(_interfaceMethod->classNameChars(), len, _comp, stackAlloc);
```

Comment:
Strictly speaking, in my previous review I was only asking about places where the result of calling `cls.classNameChars` was assigned to a `char *` variable instead of a `const char *`.  However, there's no harm in assigning the result of `cls.classNameToSignature` to a `const char *` variable as well.

Reply:
I didn't notice the difference, my bad! I've reverted not because it would be bad to assign to `const char *`, but because I feel like it has nothing to do with these changes.